lib/broccoli/ember-app.js

```diff
@@ -388,12 +385,8 @@ class EmberApp {
     @return
   */
   concatFiles(tree, options) {
-    this.project.ui.writeDeprecateLine('EmberApp.concatFiles() is deprecated. Please use the `broccoli-concat` module directly.',
-      arguments[2] === SECRET_DEPRECATION_PREVENTION_SYMBOL);
-
-    options.sourceMapConfig = this.options.sourcemaps;
-
-    return concat(tree, options);
+    this.project.ui.writeDeprecateLine('EmberApp.concatFiles() is deprecated. Please use the `broccoli-concat` module directly.');
```

Comment:
> and I presume refactors the internals to avoid calling it and instead call _concatFiles

we're actually already calling `_concatFiles()` everywhere internally. the only call to `concatFiles()` is in a unit test that tests that the deprecation message is being printed.